frontend/model/state.js

```diff
@@ -53,6 +53,13 @@ function guardedSBP (sel: string, ...data: any): any {
   return sbp(sel, ...data)
 }
 
+function checkThemeState (state) {
+  // Prevent bug if for some reason state doesn't have a theme
+  // TODO replace default from 0 to system preference if it exist
+  // Will be implemetaed in issue #844 Implement updates to the User Settings page
+  if (!state.theme) state.theme = Colors[0].theme
+}
```

Comment:
I don't understand... how is it possible for the state to not have a theme? I don't think this is necessary. Maybe it was a fluke?